crates/swc_ecma_transforms_react/src/jsx/tests.rs

```diff
@@ -1483,3 +1489,74 @@ fn integration(input: PathBuf) {
         },
     );
 }
+
+#[testing::fixture("tests/script/**/input.js")]
+fn script(input: PathBuf) {
+    let mut output = input.with_file_name("output.js");
+    if !output.exists() {
+        output = input.with_file_name("output.mjs");
```

Comment:
I think this approach is too error-prone

Reply:
Do you mean that the inject approach is too error-prone

Reply:
This line. Determining test mode based on thr existence of output

Reply:
I deleted .mjs output test for script test

Reply:
After lunch, I'll fix conflict